src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -241,11 +245,59 @@ const DismissButton = styled(Button)`
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 `;
+const ExampleEventCause = styled(Panel)`
+  overflow: hidden;
+  pointer-events: none;
+`;
+
+const ExampleRibbon = styled('div')`
+  position: absolute;
+  right: 0px;
+
+  &:after {
```

Comment:
This can just live on `ExampleEventCause` then, since I only had it in it's own div to have the overflow and pointer events off.